### ISSUE TITLE
Background and foreground colors do not have a sufficient contrast ratio.

### ISSUE THREAD

**Issue by thealpa** [thealpa/janhuelsmann.com#1]
Background and foreground colors should be adjusted to meet the WCAG success criteria.

<img width="664" alt="Screen Shot 2022-04-19 at 01 41 10" src="https://user-images.githubusercontent.com/16319455/163893572-0b562b9e-2e47-4b95-a992-d80b776b837b.png">
.
